### PR TITLE
add Go 1.16 to the build matrix

### PR DESCRIPTION
**Pull request by shogo82148**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -37,8 +37,8 @@ environment:
       GOVERSION: 115
       SQLINSTANCE: SQL2019
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
-      GOVERSION: 115
-      SQLINSTANCE: SQL2017
```
Comment (shogo82148) [inline appveyor.yml:41]: SQL2017 is already covered at https://github.com/denisenkom/go-mssqldb/blob/045585d74f9069afe2e115b6235eb043c8047043/appveyor.yml#L25
so I dropped this case.

**Comment by shogo82148** [comment]
The build fails due to https://github.com/denisenkom/go-mssqldb/issues/639
https://ci.appveyor.com/project/denisenkom/go-mssqldb/builds/38567703/job/mpadty14o33sx2re

```
panic: Not implemented [recovered]
	panic: Not implemented
goroutine 2183 [running]:
testing.tRunner.func1.2(0x64b360, 0x742b20)
	c:/go116/src/testing/testing.go:1144 +0x49f
testing.tRunner.func1(0xc000088480)
	c:/go116/src/testing/testing.go:1147 +0x695
panic(0x64b360, 0x742b20)
	c:/go116/src/runtime/panic.go:971 +0x49a
github.com/denisenkom/go-mssqldb.passthroughConn.SetWriteDeadline(...)
	C:/gopath/src/github.com/denisenkom/go-mssqldb/net.go:167
crypto/tls.(*Conn).SetWriteDeadline(...)
	c:/go116/src/crypto/tls/conn.go:150
(snip)
```

https://github.com/denisenkom/go-mssqldb/pull/642 maybe fix it.

File: appveyor.yml (hunk 1 above)
Comment (robinknaapen) [reply appveyor.yml:41]: From what I can see every matrix entry needs a `SQLINSTANCE` since there's no global setting :thinking:

**Comment by robinknaapen** [comment]
I've added Go116 pipeline to https://github.com/denisenkom/go-mssqldb/pull/642

**Comment by denisenkom** [comment]
This is done in a different PR

**Pull request closed without merging**